rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/RequestForJudgementJsonMapper.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.cmc.rpa.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.cmc.rpa.DateFormatter;
+import uk.gov.hmcts.cmc.rpa.mapper.json.NullAwareJsonObjectBuilder;
+
+import javax.json.JsonObject;
+
+@Component
+public class RequestForJudgementJsonMapper {
+    public JsonObject map(Claim claim) {
+        return new NullAwareJsonObjectBuilder()
+            .add("caseNumber", claim.getReferenceNumber())
```

Comment:
Are we sure only these 3 properties are required here? I am trying to imagine what robot would do here since the JSON does not convey any change of case details (case number, issue date and deadline remain the same)